src/assets/i18n/en.json

```diff
@@ -86,5 +86,6 @@
 	"Double click to save this period": "Double click to save this period",
 	"Types TEXT, GEO_POINT are excluded": "Types 'TEXT', 'GEO_POINT' are excluded",
 	"Area Of Interest": "Area of interest",
-	"Fill": "Polygon"
+	"Fill": "Polygon",
+	"Current AOI": "Current AOI"
```

Comment:
Not all keys from the fr.json are in the en.json. We should run an extract on the keys to translate before the final release.

Reply:
yes I agree